com.synology.CloudStationBackup.yml

```diff
@@ -0,0 +1,57 @@
+app-id: com.synology.CloudStationBackup
+default-branch: stable
+runtime: org.freedesktop.Sdk
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: synology-cloud-station-backup
+tags: [proprietary]
+finish-args:
+  - --filesystem=home
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --env=LD_LIBRARY_PATH=/app/lib
```

Comment:
Is it actually needed?

Reply:
I based these off the existing flatpak for Synology drive since it's doing a similar task. Synch/backup of files. Are you proposing rremove the --env ?

Reply:
Removed
`- --env=LD_LIBRARY_PATH=/app/lib`